.whitesource

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Why do we need this file?

Reply:
Didn't realize it was in there sorry. Its a vulnerability scanning tool that I use on my repositories.